### PR TITLE
Enable pkg repos on rhel during bootstrapping.

### PR DESCRIPTION
**Pull request by mbarnes**
This was easy enough to hack in, but I don't know how to test it short of running it on a full test infrastructure.  Any suggestions?

Also CC: @cooktheryan

### CODE REVIEW

File: src/commissaire/transport/ansibleapi.py [hunk 1]
```diff
@@ -350,6 +350,23 @@ def bootstrap(self, ip, key_file, config, oscmd):
...
+        #
+        #      See https://github.com/projectatomic/commissaire/pull/56
+        #
+        if oscmd.os_type in ('rhel', 'redhat'):
+            play_vars['commissaire_enable_pkg_repos'] = (
+                'subscription-manager repos '
+                '--enable=rhel-7-server-extras-rpms '
+                '--enable=rhel-7-server-optional-rpms')
+        else:
+            play_vars['commissaire_enable_pkg_repos'] = 'true'
```
Comment (ashcrow) [inline src/commissaire/transport/ansibleapi.py]: Is ```true``` good enough or do we need the full path like in the Atomic OSCmd?

**Comment by ashcrow** [comment]
Unittesting should be doable within the ansibleapi test file. Just checking that the command is set ONLY for ```rhel``` and ```redhat```. In terms of making sure it works in a full setting I think we will have to rely on our build testing which smoke tests the run against different OS's. Not ideal but should be good enough.

File: src/commissaire/transport/ansibleapi.py (hunk 1 above)
Comment (mbarnes) [inline src/commissaire/transport/ansibleapi.py]: I'd actually prefer the full path but I was following the example of Atomic OSCmd which just uses `true`.

**Comment by mbarnes** [comment]
Okay, I'll follow up with a unit test for that much at least.

**Comment by cooktheryan** [comment]
I really like this fix @mbarnes 

This is something we can easily modify the e2e runs to support.

**Comment by mbarnes** [comment]
:arrow_up: Rebased on master and added a fixup for the unit test.

**Comment by ashcrow** [comment]
Looks good. Squashing and merging.

**Pull request closed without merging**